.env.local.example

```diff
@@ -1,14 +1,10 @@
 ADMIN_URL_BASE=admin/
 CLOUDFLARE_API_TOKEN=
 DATABASE_URL=postgres://djangopackages:djangopackages@postgres:5432/djangopackages
-DEBUG=
-DJANGO_DEBUG=
+DJANGO_DEBUG=True
 DJANGO_MAILGUN_API_KEY=
 DJANGO_SENTRY_DSN=
-DJANGO_SETTINGS_MODULE=
-DOMAIN_NAME=
-EMAIL_HOST_USER=
-FRAMEWORK_TITLE=
+DJANGO_SETTINGS_MODULE=settings.docker
```

Comment:
Was literally about to DM you about this!

Reply:
Follow up in https://github.com/djangopackages/djangopackages/issues/915